Dockerfiles/cluster-agent/dist/templates/forwarder.tmpl

```diff
@@ -2,14 +2,16 @@
 Forwarder
 =========
 {{ if .Transactions }}
+  Transactions
+  ============
   {{- range $key, $value := .Transactions }}
-  {{$key}}: {{$value}}
+    {{$key}}: {{$value}}
   {{- end }}
   {{- if .Transactions.DroppedOnInput }}
 
-  Warning: the forwarder dropped transactions because all workers were busy.
-  You should review your network performance, and tune the forwarder_num_workers
-  and forwarder_timeout options.
+    Warning: the forwarder dropped transactions because all workers were busy.
+    You should review your network performance, and tune the forwarder_num_workers
```

Comment:
This is potentially misleading to me.  "Workers" is used in multiple places (when I first read I thought of the collector workers for checks), and probably isn't meaningful to users.

IMO we should have a shorter, more generic message with a link to a complete KB/manual page?